dev-docs/bidders/stroeerCore.md

```diff
@@ -11,9 +11,11 @@ usp_supported: false
 safeframes_ok: true
 prebid_member: false
 pbjs: true
-pbs: false
+pbs: true
 gvl_id: 136
+pbs_app_supported: true
 deals_supported: true
+userIds: criteo, id5Id, netId, pubCommonId, sharedId
```

Comment:
Thanks for updating the meta info. While you are at it, could you add the flollowing fields as well?

- *multiformat_supported*: will-bid-on-any, will-bid-on-one, will-not-bid
- *floors_supported*: true/false
- *fpd_supported*: true/false

:smile: